uaclient/entitlements/tests/test_cc.py

```diff
@@ -168,7 +160,7 @@ def exists(path):
             mock.call(
                 "/etc/apt/sources.list.d/ubuntu-cc-eal-xenial.list",
                 "http://CC",
-                "TOKEN",
+                "%s-token" % entitlement.name,
```

Comment:
A nit, but `.format` here would be more consistent with the rest of the codebase.